src/docfx/config/Config.cs

```diff
@@ -122,6 +122,11 @@ internal sealed class Config
         /// </summary>
         public readonly bool WarningsAsErrors;
 
+        /// <summary>
+        /// Gets the urls to restore xref map
+        /// </summary>
+        public readonly List<string> XrefMapUrls = new List<string>();
```

Comment:
just `xref` like in [v2](https://dotnet.github.io/docfx/tutorial/docfx.exe_user_manual.html)